### PR TITLE
LRAC-12557, LRAC-12475, LRAC-12564

### PR DESCRIPTION
**Pull request by ilzamcmed**
https://issues.liferay.com/browse/LRAC-12557
https://issues.liferay.com/browse/LRAC-12475
https://issues.liferay.com/browse/LRAC-12564

### CODE REVIEW

**Comment by ilzamcmed** [comment]
ci:test:relevant

**Comment by ilzamcmed** [comment]
ci:test:sf

**Comment by ilzamcmed** [comment]
ci:test:relevant

**Comment by ilzamcmed** [comment]
ci:test:relevant

**Comment by ilzamcmed** [comment]
ci:test:sf

**Comment by ilzamcmed** [comment]
ci:test:relevant

File: modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -60,3 +60,7 @@
 	right: 0;
 	top: 0;
 }
+
+.c-empty-state-animation {
```
Comment (fmarcio) [inline modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/css/main.scss:64]: hey @ilzamcmed could you please sort this? Thanks!

**Review by fmarcio — CHANGES_REQUESTED** [review]

**Review by interaminense — APPROVED** [review]
LGTM, @ilzamcmed! Just take a look at Marcio's comment

File: modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/css/main.scss (hunk 1 above)
Comment (ilzamcmed) [reply modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/css/main.scss]: Fixed! Thanks @fmarcio!

**Comment by ilzamcmed** [comment]
ci:test:sf

**Comment by ilzamcmed** [comment]
ci:test:relevant

**Review by fmarcio — APPROVED** [review]
LGTM! :rocket:

**Comment by interaminense** [comment]
ci:forward

**Pull request closed without merging**